server/api.go

```diff
@@ -805,7 +805,7 @@ func (p *Plugin) createIssueComment(w http.ResponseWriter, r *http.Request) {
 
 	siteUrl := api.GetConfig().ServiceSettings.SiteURL
 
-	permalink := getPermaLink(*siteUrl, req.PostId, req.CurrentTeam)
```

Comment:
Since we do not use it here, do we need passing around the current team at all?

Reply:
Good catch. Done in 8dad1cd5d89a93d96aca6a44e7d7f02291a00fba